conda/environments/cudf_dev_cuda11.5.yml

```diff
@@ -76,3 +78,8 @@ dependencies:
       - git+https://github.com/python-streamz/streamz.git@master
       - pyorc
   - ptxcompiler  # [linux64]
+  - gcc_linux-64=9.3.0 # [linux64]
+  - sysroot_linux-64==2.17 # [linux64]
+  # Un-comment following lines for ARM specific packages.
+  # - gcc_linux-aarch64=9.3.0 # [aarch64]
+  # - sysroot_linux-aarch64==2.17 # [aarch64]
```

Comment:
I had to comment out the arm specific package due to lack of selectors: https://github.com/conda/conda/issues/8089

The only way I could see using this file is un-commenting the last two lines when working on an arm-dev machine. Since this is used only by developers I think this is okay. 

cc: @jakirkham if you have any inputs on this.

Reply:
Not really. The only other option atm is two different files, which might not be worth it

Reply:
Yea, let's leave it as is for now. If we see a growing number of arm-specific packages we can be splitting it into two files.